src/codegen/index.js

```diff
@@ -1,11 +1,16 @@
 const babel = require('prettier/parser-babel')
+const evaluateVariable = require('../render/evaluate')
 
 const BUILD = Symbol('build')
 const UNQUOTED = Symbol('unquoted')
+const VARIABLE_REGEX = /^\$\{(\w+)\}$/
```

Comment:
Looks like there is already an expression for identifying variables, it seems to be used throughout the codebase. Lets use that same expression `src/expression.js`